lib/sql_parser.mly

```diff
@@ -156,7 +156,7 @@ statement: CREATE ioption(temporary) TABLE ioption(if_not_exists) name=table_nam
                 Update (table,ss,w,o,lim)
               }
          /* http://dev.mysql.com/doc/refman/5.1/en/update.html multi-table syntax */
-         | update_cmd tables=commas(table_list) SET ss=commas(set_column) w=where?
+         | update_cmd tables=commas(table_list) SET ss=commas(set_column) w=where? loption(order) loption(limit)
```

Comment:
It seems to work even without passing `order` and `limit` to `UpdateMulti` (and unlike for `Update` case above). Is it expected?